drivers/ntc/ntc_tcp.c

```diff
@@ -827,5 +845,6 @@ static void __exit ntc_tcp_exit(void)
 		if (ntc_tcp_devices[i])
 			ntc_tcp_destroy(ntc_tcp_devices[i]);
 }
-module_exit(ntc_tcp_exit);
 
+
```

Comment:
Better to just leave module_exit() where it was.  It has nothing to do with the rest of the patch.